src/service/xhr-impl.js

```diff
@@ -534,6 +535,14 @@ class FetchResponseHeaders {
   get(name) {
     return this.xhr_.getResponseHeader(name);
   }
+
+  /**
+   * @param {string} name
+   * @return {boolean}
+   */
+  has(name) {
+    return !!this.xhr_.getResponseHeader(name);
```

Comment:
[Quoth MDN:](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getResponseHeader)

> Starting from Firefox 49, empty headers are returned as empty strings in case the preference network.http.keep_empty_response_headers_as_empty_string is set to true, defaulting to false. Before Firefox 49 empty headers had been ignored. Since Firefox 50 the preference defaults to true.